pinc/page_controls.inc

```diff
@@ -84,8 +84,6 @@ function get_proofreading_interface_data_js()
     $font_data = [
         "faces" => get_font_styles(),
         "faceFamilies" => get_full_font_families(),
-        "sizes" => get_font_sizes(),
```

Comment:
If you do:
```
"sizes" => [ get_font_sizes() ],
"sizeFamilies" => [ get_font_sizes_css() ],
```
The dicts inside the arrays will be sorted like they are in PHP.

Reply:
Or we could just transfer the sizes without the indices.